### PR TITLE
DT-1273 Initial Skeleton

### PR DESCRIPTION
**Pull request by mikehalmamoj**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -8,5 +8,15 @@ configurations {
 }
```
Comment (JamieDraperUK) [inline build.gradle.kts:8]: Should the it come with the Kotlin lint module and the .editorconfig file?

**Comment by JamieDraperUK** [comment]
Does it need an example health endpoint?

**Review by JamieDraperUK — APPROVED** [review]
LGTM

File: build.gradle.kts (hunk 1 above)
Comment (mikehalmamoj) [reply build.gradle.kts:8]: A very good question, I wasn't sure if we are definitely adopting ktlint or not.  I will find out and stick it in a future PR, thanks.

**Pull request merged**